### DISCUSSION TITLE
How do you use existing world files?

### DISCUSSION THREAD

**Discussion by cpbreton** [mbround18/valheim-docker#128] [Q&A] [answered]
When mounting volumes how do I get the container to see and use my existing world files?

**Comment by mbround18** [comment, marked as answer]
Copy the existing world files into the mounted location. (Copy instead of move just in case you have to do it again.)

For example, lets say my world files are at 
````
/home/myuser/valheim-saves/worlds
````

You will want to copy the contents of the worlds directory into your mounted server location
````
cp -R /home/myuser/valheim-saves/worlds/*  /path/to/where/you/mount/saves/worlds/
````

Then when you start your Valheim server, make sure the world name matches what the world name in the file system is.

**Reply by cpbreton** [reply]
![docker](https://user-images.githubusercontent.com/77699738/108263064-c22e0a00-7133-11eb-8d97-3c4f73882b48.PNG)

Does this look right for those volume mappings? Like if my world files are in a folder called "worlds" in that saves folder, that should work?

**Reply by cpbreton** [reply]
Ok, I might be doing something wrong but I got it to work by mounting a folder to /home/steam/.config/unity3d/IronGate/Valheim/worlds

**Reply by Flightkick** [reply]
@cpbreton You were mounting your host volume `C:\users\...\valheim\saves` to the container volume `/valheim/saves` which is not used.
The save files are located in the `/home/steam/.config/unity3d/IronGate/Valheim` in the container, hence it worked after you corrected the mappings.

**Reply by cpbreton** [reply]
Yeah, I didn't know how to get my world files into the system or the file structure of the container. Glad it's working now though. Thank you @mbround18 for this awesome thing.
